docs/sdk-and-tools/notifier.md

```diff
@@ -30,11 +30,11 @@ Set up at least one observer for each shard in order to handle all the events in
 ![img](/technology/notifier-overview.png)
 
 In the figure above:
-- The observer nodes will push block events to Notifier instance, via HTTP POST requests. There are several endpoints for this:
-    - `/events/push` (POST) -> it will handle all events for each round
-    - `/events/revert` (POST) -> if there is a reverted block, the event will be
+- The observer nodes will push block events to Notifier instance, via WebSocket or HTTP requests. There are several endpoints/topics for this:
+    - `block events` -> it will handle all events for each round
+    - `revert events` -> if there is a reverted block, the event will be
       pushed on this route
-    - `/events/finalized` (POST) -> when the block has been finalized, the events
+    - `finalized events` -> when the block has been finalized, the events
```

Comment:
replace `Websockets` with `Websocket` on L40 and other occurrences

Reply:
changed